lib/miq_automation_engine/engine/miq_ae_method.rb

```diff
@@ -289,5 +255,56 @@ def self.invoke_inline_ruby(aem, obj, inputs)
         end
       end
     end
+
+    def self.run_method(cmd)
+      require 'open4'
+      rc = nil
+      final_stderr = []
+      threads = []
+      method_pid = nil
+      begin
+        require 'open4'
+        status = Open4.popen4(*cmd) do |pid, stdin, stdout, stderr|
+          method_pid = pid
+          yield stdin if block_given?
+          stdin.close
+          threads << Thread.new do
+            stdout.each_line { |msg| $miq_ae_logger.info "Method STDOUT: #{msg.strip}" }
+          end
+          threads << Thread.new do
+            stderr.each_line do |msg|
+              msg = msg.strip
```

Comment:
I know the existing code uses `strip`.. but `chomp` is really what's called for. Any leading whitespace seems valuable.